apple-pie/styles/style.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+
+body {
+    font-family: Roboto;
+    margin: 0;
+    padding: 0;
+}
+
+header {
+    background-image: url("/apple-pie/images/apple-pie.jpg");
```

Comment:
When you write path here, remember that you need to go to the image from the file where you define the path , in this case from style.css file.
 style.css ->styles -> images -> apple-pie.jpg

 background-image: url("../images/apple-pie.jpg")